In here the exception handling is done 
with unittest and pylint
